test/integration/clinicians/clinicians-all.js

```diff
@@ -286,5 +291,25 @@ context('clinicians list', function() {
       .should('have.length', 1)
       .first()
       .should('contain', 'Employee');
+
+    cy
+      .get('[data-nav-content-region]')
+      .find('[data-worklists-region]')
+      .find('.app-nav__link')
+      .first()
+      .click()
+      .go('back');
+
+    cy
+      .get('.list-page__header')
+      .find('[data-search-region] .js-input:disabled')
+      .should('have.attr', 'placeholder', 'Find in List...');
+
+    cy
+      .wait('@routeClinicians');
+
+    cy
+      .get('.list-page__header')
+      .find('[data-search-region] .js-input:not([disabled])');
```

Comment:
This addition to the test simply navigates to a new page, comes back to the clinicians page, and verifies that the `Find in list...` input is disabled/enabled properly.